Exiled.Events/Patches/Events/Warhead/Detonated.cs

```diff
@@ -29,6 +28,45 @@ private static IEnumerable<CodeInstruction> Transpiler(IEnumerable<CodeInstructi
         {
             List<CodeInstruction> newInstructions = ListPool<CodeInstruction>.Pool.Get(instructions);
 
+            Label continueLabel = generator.DefineLabel();
+            LocalBuilder ev = generator.DeclareLocal(typeof(DetonatingEventArgs));
+
+            newInstructions[0].WithLabels(continueLabel);
+
+            newInstructions.InsertRange(
+                0,
+                new CodeInstruction[]
+                {
+                    // DetonatingEventArgs ev = new();
+                    new(OpCodes.Newobj, GetDeclaredConstructors(typeof(DetonatingEventArgs))[0]),
+                    new(OpCodes.Stloc, ev.LocalIndex),
+
+                    // Handlers.Warhead.OnDetonating(ev);
+                    new(OpCodes.Ldloc_S, ev.LocalIndex),
```

Comment:
Another thing that doesn't matter so much, it is better to prefer using short forms of op codes (with `_S` postfix).
`Stloc` can be replaced by `Stloc_S`

(But you can ignore it rly, this is the most minimal optimization that can be)